packages/statemanager/src/stateManager.ts

```diff
@@ -1017,6 +1024,15 @@ export class DefaultStateManager implements EVMStateManagerInterface {
    The object will also contain `nextKey`, the next (hashed) storage key after the range included in `storage`.
    */
   async dumpStorageRange(address: Address, startKey: bigint, limit: number): Promise<StorageRange> {
+    function nibblestoBytes(arr: Nibbles): Uint8Array {
```

Comment:
I don't love that we're duplicating code here that already exists in `trie` but I get why we're doing it.  It's fine for purposes of this PR but a takeaway is to clean up our `trie` utility methods and then come back and remove this.

Reply:
Nevermind, found the appropriate function in `trie` and have imported it instead of duplicating the code here.